DOCS/man/mpv.rst

```diff
@@ -499,32 +499,53 @@ console controls. (Which makes it suitable for playing data piped to stdin.)
 The special argument ``--`` can be used to stop mpv from interpreting the
 following arguments as options.
 
-When using the client API, you should strictly avoid using ``mpv_command_string``
+For paths passed to mpv CLI options, the situation is further complicated by the
+need to escape special characters. To work around this, the path can instead be
+wrapped in the "fixed-length" syntax, e.g. ``%n%string_of_length_n`` (see above).
+
+When using the libmpv API, you should strictly avoid using ``mpv_command_string``
 for invoking the ``loadfile`` command, and instead prefer e.g. ``mpv_command``
 to avoid the need for filename escaping.
 
-For paths passed to suboptions, the situation is further complicated by the
-need to escape special characters. To work this around, the path can be
-additionally wrapped in the fixed-length syntax, e.g. ``%n%string_of_length_n``
-(see above).
+The same applies when you're using the scripting API, where you should avoid using
+``mp.command``, and instead prefer using the "separate parameters" variants (such
+as ``mp.commandv`` or ``mp.command_native``), so that the filenames will always be
+correctly interpreted.
+
+Some mpv options will interpret special meanings for paths starting with ``~``,
+making it easy to dynamically find special directories, such as referring to the
+current user's home directory or the mpv configuration directory.
+
+When using the special ``~`` prefix, there must always be a trailing ``/`` after
+the special path prefix. In other words, ``~`` doesn't work, but ``~/`` will work.
+
+The following special paths/keywords are currently recognized:
+
+.. warning::
+
+    Beware that if ``--no-config`` is used, all of the "config directory"-based
+    paths (``~~/``, ``~~home/`` and ``~~global/``) will resolve to empty strings.
+
+    This means that commands such as ``expand-path "~~home/"`` will return an
+    empty string, and ``expand-path "~~home/foo/bar"`` will only return ``foo/bar``
+    (without any path prefix), which might not be what you expected.
+
+    Furthermore, certain commands such as ``mp.find_config_file("mpv.conf")``
```

Comment:
This is internal API, why is this mentioned here? Why external scripts uses it?

Reply:
I saw it in `lua.c` and in various internal `.lua` scripts and am using it myself to find files in the config-dirs. It does the right thing: Searches all config dirs for the given path.

I did not find any alternatives that can search all config dirs from user scripts.

Reply:
Anyway, I'll remove the mention of the specific command and just say that any commands that search in config dirs will fail.